zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -523,6 +537,21 @@ trait Sql {
     )(implicit ev: B <:< Integer): Expr[F :||: F2, A1, Integer] =
       Expr.Binary(self.widen[Integer], that, BinaryOp.OrBit)
 
+    def ~[A1 <: A]()(implicit ev: B <:< Integer): Expr.Unary[F, A1, Integer] =
+      Expr.Unary(self.widen[Integer], UnaryOp.NotBit)
+
+    def -[A1 <: A]()(implicit ev: B <:< Integer): Expr.Unary[F, A1, Integer] =
+      Expr.Unary(self.widen[Integer], UnaryOp.NotNum)
+
+    def not[A1 <: A]()(implicit ev: B <:< Boolean): Expr.Unary[F, A1, Boolean] =
+      Expr.Unary(self.widen[Boolean], UnaryOp.NotBool)
+
+    def isNull[A1 <: A](): Expr.Unary[F, A1, Any] =
```

Comment:
There is no need for empty parentheses. In Scala, empty parens = side-effect. But ZIO SQL is a pure functional library and has no side effects.

Reply:
Very good point. Removed ()